src/main/java/org/gridsuite/filter/server/dto/NumericalFilter.java

```diff
@@ -9,13 +9,15 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import org.gridsuite.filter.server.utils.RangeType;
 
 /**
  * @author Jacques Borsenberger <jacques.borsenberger at rte-france.com>
  */
 
 @Getter
+@NoArgsConstructor
```

Comment:
is this change concern filter renaming ?

Reply:
needed for test (ObjectMapper needs it)